notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>"...temperature over the continent..." Which continet, or is it for each continet? Slightly unclear wording</p><p> </p><p>Consider referencing a paper where I can find out more about the OBS blending effects</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Done.</p>

Reply:
closing conversation, resolved